### PR TITLE
New Salt API Cheatsheet

### PR DESCRIPTION
**Pull request by k00laidzone**
Added the salt-api cheat sheet

---
https://duck.co/ia/view/salt_api_cheat_sheet

### CODE REVIEW

**Comment by daxtheduck** [comment]
@k00laidzone Hey!
        
Thanks for taking the time to contribute! We really appreciate it.

We work closely with every contributor to make Instant Answers the best they can be, so we appreciate your patience as we look over your code. From here, the process usually goes like this:

1. Pull Request is reviewed by the DuckDuckGo staff and community
2. Staff and community will leave feedback with any necessary updates to the function or design.
3. Once you've made any necessary corrections, then your Instant Answer will be merged and deployed live on DuckDuckGo!

While the team and the community review your submission, we'd like to make sure you look good! If you'd like a free DuckDuckGo tshirt, please send over the following info to merch@duckduckgo.com, and be sure to put "Dev Merch" in the subject line:

Name:
Street Address:
City:
State / Province:
Country:
Phone (Non-US Orders only): 
T-shirt size:
GitHub ID:
Link to this Pull Request:

If you have any questions along the way, feel free to ask them here. Our staff and community are also available on Slack to answer any questions you may have. If you'd like to join us there please send an email to <quackslack@duckduckgo.com>.

>More Info: https://duck.co/duckduckhack/submission_and_review

Thanks!

**Comment by daxtheduck** [comment]
Metadata post coming soon

**Comment by moollaza** [comment]
@k00laidzone thanks a lot for contributing.

Can you please remove 1 one the cheat sheets from this Pull Request, and re-submit it from a separate PR?

The reason being that we have some code on our Community Platform that uses the PR to determine which files belong with which Instant Answers.

Once that's done, we'll be able to review these Cheat Sheets and get them deployed to Beta for testing.

File: share/goodie/cheat_sheets/json/salt api.json [hunk 1]
```diff
@@ -0,0 +1,56 @@
+{
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/salt api.json]: For this file, please change the filename to `sale-api.json` (no spaces)

File: share/goodie/cheat_sheets/json/salt api.json [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+    "id": "saltapi_cheat_sheet",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/salt api.json]: the `id` will need to match the filename: `salt_api_cheat_sheet`

**Comment by moollaza** [comment]
Also, can you please fill out the [pull request template](https://github.com/duckduckgo/zeroclickinfo-goodies/blob/master/pull_request_template_cheat_sheet.md) and edit your initial comment.

Thanks!

BTW, If you need help our @duckduckgo/duckduckhack-mentors are available to assist you as well

File: share/goodie/cheat_sheets/json/salt-api.json [hunk 3]
```diff
@@ -0,0 +1,48 @@
+{
+    "id": "salt_api_cheat_sheet",
+    "name": "Salt API",
+    "description": "Common commands for SaltStack API",
+    "metadata": {
+        "sourceName": "SaltStack Website",
```
Comment (jophab) [inline share/goodie/cheat_sheets/json/salt-api.json]: @k00laidzone Lets remove website from source name

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=)
                
                [Mobile screenshot](https://ia-screenshots.s3.amazonaws.com/salt_api_cheat_sheet_mobile.png)

File: share/goodie/cheat_sheets/json/salt-api.json [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+        "sourceUrl": "https://docs.saltstack.com/en/latest/ref/cli/"
+    },
+    "template_type": "terminal",
+    "section_order": [
+        "Options",
+        "Logging options"
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/salt-api.json]: This needs to match the section below: `Logging Oprtions`

**Comment by moollaza** [comment]
@k00laidzone this LGTM :+1: (there's a couple minor fixes I'll make for you)

Thanks a lot for contributing (we really appreciate it :smiley:). We invite you to stick around and comment/help on other PR's & Issues or even submit an Instant Answer!

**Pull request merged**